Language/Functions/Bits and Bytes/bitWrite.adoc

```diff
@@ -59,6 +59,7 @@ void setup() {
   Serial.println(x, BIN); // 10000000
   bitWrite(x, 0, 1);  // write 1 to the least significant bit of x
   Serial.println(x, BIN); // 10000001
+  Serial.println(bitWrite(x, 1, 1), BIN); // 10000011
```

Comment:
```suggestion
```

The example is meant to be a demonstration of the basic usage of the function. Even though it is indisputable that the "return" of the macro must be documented correctly, I don't think it needs to be demonstrated since this is not a common use pattern for `bitWrite`.